x-pack/plugins/upgrade_assistant/public/application/components/kibana_deprecations/kibana_deprecations.tsx

```diff
@@ -50,25 +50,25 @@ const i18nTexts = {
     defaultMessage: 'Kibana',
   }),
   isLoading: i18n.translate('xpack.upgradeAssistant.kibanaDeprecations.loadingText', {
-    defaultMessage: 'Loading deprecations…',
+    defaultMessage: 'Loading deprecation issues…',
   }),
   kibanaDeprecationErrorTitle: i18n.translate(
     'xpack.upgradeAssistant.kibanaDeprecations.kibanaDeprecationErrorTitle',
     {
-      defaultMessage: 'Deprecation warnings may be incomplete',
+      defaultMessage: 'Deprecation issues may be incomplete',
     }
   ),
   getKibanaDeprecationErrorDescription: (pluginIds: string[]) =>
     i18n.translate('xpack.upgradeAssistant.kibanaDeprecations.kibanaDeprecationErrorDescription', {
       defaultMessage:
-        'Failed to get deprecation warnings for {pluginCount, plural, one {this plugin} other {these plugins}}: {pluginIds}. Check the Kibana server logs for more details.',
+        'Failed to get deprecation issues for {pluginCount, plural, one {this plugin} other {these plugins}}: {pluginIds}. Check the Kibana server logs for more details.',
```

Comment:
I'd say either "for details" or "for more information". 
```suggestion
        'Failed to get deprecation issues for {pluginCount, plural, one {this plugin} other {these plugins}}: {pluginIds}. Check the Kibana server logs for more information.',
```